src/state/routing/slice.ts

```diff
@@ -143,6 +152,8 @@ export const routingApi = createApi({
               tokenOut: tokenOutAddress,
               amount,
               type,
+              // if forceUniswapXOn is not ON, then use the backend's default value
+              useUniswapX: forceUniswapXOn || undefined,
```

Comment:
nit: I think this would be safer (for future maintenance) if this were pulled into a separate typed variable:

```
const useUniswapX: true | undefined = forceUniswapXOn || undefined
```

Then it's `tsc` checking the logic instead of a human.

Reply:
not terribly concerned about future maintenance as we'll rip this out as soon as its deployed :D. but good to keep in mind for next time!